rsts/user/tasktypes/presto.rst

```diff
@@ -0,0 +1,127 @@
+.. _presto-task-type:
+
+=============
+Presto Tasks
```

Comment:
Do we only support "querying" through Presto? is there an equivalent to "write to hive" task?

Should we elaborate that the query should only be the select statement since the "write outputs to a file" step is done by the plugin... but write that in a more humanly readable form ?

Reply:
Makes sense. I  added a new section called `Flyte Presto Task` where it talks about some of the internals of how the task works.